### PR TITLE
Added mapping of time limit and memory limit for submission type in problem

### PR DESCRIPTION
**Pull request by shsimeonova**
Closes https://github.com/SoftUni-Internal/exam-systems-issues/issues/1079

### CODE REVIEW

File: Services/UI/OJS.Services.Ui.Models/Contests/ContestProblemServiceModel.cs [hunk 1]
```diff
@@ -83,13 +73,13 @@ public void RegisterMappings(IProfileExpression configuration) =>
...
                     d => d.AllowedSubmissionTypes,
-                    opt => opt.MapFrom(s => s.SubmissionTypesInProblems.Select(st => st.SubmissionType)))
+                    opt => opt.MapFrom(s => s.SubmissionTypesInProblems))
                 .ForMember(
                     d => d.Points,
                     opt => opt.MapFrom(s => 0))
                 .ForMember(
                     d => d.MemoryLimit,
-                    opt => opt.MapFrom(s => (double)s.MemoryLimit))
+                    opt => opt.MapFrom(s => (double?)s.MemoryLimit / 1024 / 1024))
```
Comment (gogo4ds) [inline Services/UI/OJS.Services.Ui.Models/Contests/ContestProblemServiceModel.cs:82]: Can we do the conversion on the front end?

File: Services/UI/OJS.Services.Ui.Models/Contests/ContestProblemServiceModel.cs [hunk 2]
```diff
@@ -98,6 +88,6 @@ public void RegisterMappings(IProfileExpression configuration) =>
                     opt => opt.MapFrom(s => s.Resources.OrderBy(x => x.OrderBy)))
                 .ForMember(
                     d => d.TimeLimit,
-                    opt => opt.MapFrom(s => (double)s.TimeLimit));
+                    opt => opt.MapFrom(s => (double?)s.TimeLimit / 1000));
```
Comment (gogo4ds) [inline Services/UI/OJS.Services.Ui.Models/Contests/ContestProblemServiceModel.cs:91]: And here

File: Services/UI/OJS.Services.Ui.Models/SubmissionTypes/SubmissionTypeServiceModel.cs [hunk 3]
```diff
@@ -29,6 +36,37 @@ public void RegisterMappings(IProfileExpression configuration) =>
...
+                    opt => opt.MapFrom(s => s.SubmissionType.AllowBinaryFilesUpload))
+                .ForMember(
+                    d => d.IsSelectedByDefault,
+                    opt => opt.MapFrom(s => s.SubmissionType.IsSelectedByDefault))
+                .ForMember(
+                    d => d.TimeLimit,
+                    opt => opt.MapFrom(s => s.TimeLimit != null ? (double?)s.TimeLimit / 1000 : null))
+                .ForMember(
+                    d => d.MemoryLimit,
+                    opt => opt.MapFrom(s => s.MemoryLimit != null ? (double?)s.MemoryLimit / 1024 / 1024 : null))
```
Comment (gogo4ds) [inline Services/UI/OJS.Services.Ui.Models/SubmissionTypes/SubmissionTypeServiceModel.cs:61]: And here

**Review by gogo4ds — CHANGES_REQUESTED** [review]

File: Services/UI/OJS.Services.Ui.Models/SubmissionTypes/SubmissionTypeServiceModel.cs (hunk 3 above)
Comment (shsimeonova) [reply Services/UI/OJS.Services.Ui.Models/SubmissionTypes/SubmissionTypeServiceModel.cs]: Fixed

File: Services/UI/OJS.Services.Ui.Models/Contests/ContestProblemServiceModel.cs (hunk 2 above)
Comment (shsimeonova) [reply Services/UI/OJS.Services.Ui.Models/Contests/ContestProblemServiceModel.cs]: Fixed

File: Services/UI/OJS.Services.Ui.Models/Contests/ContestProblemServiceModel.cs (hunk 1 above)
Comment (shsimeonova) [reply Services/UI/OJS.Services.Ui.Models/Contests/ContestProblemServiceModel.cs]: Fixed

**Review by gogo4ds — APPROVED** [review]

**Pull request merged**